packages/office-ui-fabric-react/package.json

```diff
@@ -47,6 +48,7 @@
     "node-sass": "^4.5.0",
     "office-ui-fabric-core": ">=5.0.0 <6.0.0",
     "postcss-loader": "^1.3.3",
+    "prop-types": "^15.5.9",
```

Comment:
Let's not confuse types and dependencies. `react` itself is registered as a `peerDependency`. The guidance for `prop-types` https://www.npmjs.com/package/prop-types recommends adding `prop-types` as a dependency.